src/LocalDelivery/Resources/views/page/form.html.twig

```diff
@@ -0,0 +1,78 @@
+{% sw_extends '@Storefront/storefront/base.html.twig' %}
+
+{% block base_content %}
+    <h1>Register as DeliveryBoy</h1>
+
+    {% if errors %}
+        <div class="error-message-container alert alert-error">
+            <ul>
+                {% for error in errors %}
+                    <li>{{ error }}</li>
+                {% endfor %}
+            </ul>
+        </div>
+    {% endif %}
+
+    <form action="{{ path('delivery.boy.register') }}"
+          method="post"
+          data-form-csrf-handler="true">
+
+        {{ sw_csrf('delivery.boy.register') }}
+
+        <div class="form-row">
+            <label>Title
```

Comment:
We have snippets in src/Portal/Resources/snippets feel free to use them 😉

Reply:
in next pr